contracts/mocks/CountersImpl.sol

```diff
@@ -21,6 +22,22 @@ contract CountersImpl {
         _counter.decrement();
     }
 
+    function getAndIncrement() public {
+        helper = _counter.getAndIncrement();
```

Comment:
We normally use events to inspect the return values in this case. If you could do that it would be appreciated.

Reply:
done